data/database.js

```diff
@@ -38,63 +38,114 @@ class Database {
     }
   }
 
-  async getAudioFileLocation(articleId, audioType) {
-    logger.info(`getAudioFileLocation for ${articleId}/${audioType}`);
-    const fileLocation = await AudioFileLocation.get({ item_id: articleId });
-    if (fileLocation) {
-      if (audioType === 'full' && fileLocation.full_audio_location) {
-        return fileLocation.full_audio_location;
-      } else if (
-        audioType === 'summary' &&
-        fileLocation.summary_audio_location
-      ) {
-        return fileLocation.summary_audio_location;
-      }
-    }
-    return '';
+  async getAudioFileLocation(articleId, type, voice) {
```

Comment:
Is it valuable to include voice in these lookup function? do we need to maintain multiple files for the same article but a different voice? if users can choose their own voice as a preference, then I suppose the answer is yes

Reply:
Yeah, it may not be that useful.  Just trying to keep it as flexible as can for now.